app/controllers/comments_controller.rb

```diff
@@ -15,9 +15,21 @@ def create
     end
```

Comment:
- [ ] When I log in using another account and I try to add a comment for another's user post, It shows me the following error:

<img width="758" alt="image" src="https://user-images.githubusercontent.com/77190203/169036071-85726eb2-c7b0-4d9f-be49-a08695e8d321.png">

- Please make sure that the current user is able to like and add comments to other users' posts.

Reply:
@medaminedev66  Thanks I have implemented as suggested.